[FIX] account_edi_ubl_cii/bis3: remove deprecated EAS code for Sweden

JordiMForgeFlow opened the pull request:
remove deprecated EAS code for Sweden. The code is no longer part of the valid list of codes.

See https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/ and https://docs.peppol.eu/edelivery/codelists/

Also, in the endpoint the Swedish organization number must be used, which is taken from the VAT by removing the country code and the last two digits. You can check it out here: https://organisationsnummer.dev/

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/153440).

JulienVR commented on addons/account_edi_ubl_bis3/models/account_edi_format.py:
```diff
@@ -42,7 +42,7 @@
     'VA': 9953,
 
-    'SE': 9955,
+    'SE': '0007',
```
I don't think this module is still used but it doesn't hurt I guess

JulienVR reviewed (approved):
I'll ask @smetl or @h4818 to r+ :) Thank you !

h4818 commented:
@JordiMForgeFlow could you please rebase?

JordiMForgeFlow commented:
@JulienVR @h4818 I have rebased and also added a fix to use correctly the swedish organization number in the endpoint, which is not the same format as the VAT.

I have reached out for support to Peppol:
![image](https://github.com/odoo/odoo/assets/71635103/b4e2d50f-f85a-463e-a6f1-23568ba61869)

JulienVR commented on addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py:
```diff
@@ -138,6 +138,8 @@ def _get_partner_party_vals(self, partner, role):
             })
         if partner.country_id.code == "LU" and 'l10n_lu_peppol_identifier' in partner._fields and partner.l10n_lu_peppol_identifier:
             vals['endpoint_id'] = partner.l10n_lu_peppol_identifier
+        if partner.country_id.code == "SE" and partner.vat:
+            vals['endpoint_id'] = partner.vat.replace("SE", "")[:-2]
```
Don't forget to remove this in 16.2, as the `peppol_endpoint` field exists (and can contain a value different from the vat).

smetl commented:
@robodoo r+

smetl commented:
@robodoo rebase-ff

robodoo commented:
Merge method set to rebase and fast-forward.
